src/Models/AuthenticationLogRecord.php

```diff
@@ -20,27 +20,28 @@ public function authenticatable()
         return $this->morphTo();
     }
 
-    public static function createWithConfigFilters(array $data)
+    private static function guardIsAccepted(array $data)
     {
-        // TODO should find a nicer way to write the to avoid copy and pasting
         if(!array_key_exists('guard', $data)) {
+            return true;
+        }
 
-            if(array_key_exists('credentials', $data)) {
-                $data['credentials'] = ConfigManager::omitCredentials($data['credentials']);
-            }
-            self::create(ConfigManager::omitFields($data));
-
+        else if(ConfigManager::guardAccepted($data['guard'])) {
```

Comment:
Not a fan of this whitespace - also as a PHP technicality you should use `elseif` rather than `else if`; theoretically this will give you a (albeit negligable) performance boost